dtmi/azureiot/phone_as_a_device/phoneasadevice-1.json

```diff
@@ -0,0 +1,175 @@
+            "name": "accelerometer",
+            "schema": "dtmi:azureiot:Phone_As_A_Device:Sensors:Accelerometer;1"
+        },
+        {
+            "@type": "Component",
+            "name": "battery",
+            "schema": "dtmi:azureiot:Phone_As_A_Device:Sensors:Battery;1"
+        },
+        {
+            "@type": "Component",
+            "name": "location",
+            "schema": "dtmi:azureiot:Phone_As_A_Device:Sensors:Location;1"
+        },
+        {
+            "@type": "Component",
+            "name": "magnet",
+            "schema": "dtmi:azureiot:Phone_As_A_Device:Sensors:Magnet;1"
+        },
+        {
+            "@type": "Component",
+            "name": "pressure",
+            "schema": "dtmi:azureiot:Phone_As_A_Device:Sensors:Pressure;1"
+        },
+        {
+            "@type": "Component",
+            "displayName": {
+                "en": "Device information"
+            },
+            "name": "device_info",
+            "schema": "dtmi:azure:DeviceManagement:DeviceInformation;1"
+        },
+        {
+            "@type": "Command",
+            "commandType": "synchronous",
+            "displayName": {
+                "en": "LightOn"
+            },
+            "name": "lightOn",
+            "request": {
+                "@type": "CommandPayload",
+                "displayName": {
+                    "en": "Parameters"
+                },
+                "name": "parameters",
+                "schema": {
+                    "@type": "Object",
+                    "displayName": {
+                        "en": "Object"
+                    },
+                    "fields": [
+                        {
+                            "displayName": {
+                                "en": "Duration"
+                            },
+                            "name": "duration",
+                            "schema": "integer"
+                        },
+                        {
+                            "displayName": {
+                                "en": "Pulses interval"
+                            },
+                            "name": "delay",
+                            "schema": "integer"
+                        },
+                        {
+                            "displayName": {
+                                "en": "Pulses"
+                            },
+                            "name": "pulses",
+                            "schema": "integer"
+                        }
+                    ]
+                }
+            }
+        },
+        {
+            "@type": "Command",
+            "commandType": "asynchronous",
+            "displayName": {
+                "en": "Clear Connectivity"
+            },
+            "name": "clearConnectivity"
+        },
+        {
+            "@type": "Property",
+            "displayName": {
+                "en": "telemetry Interval (seconds)"
+            },
+            "name": "telemetryInterval",
```

Comment:
since we are already an enum for the theme? can we keep this int? to allow the user to specify how many seconds they want?
If so, consider adding a semantic type

Reply:
In the app, the values are not free-form. Using an int here and it not matching values allowed in app would result in app crashing.